integration_tests/loop_06.py

```diff
@@ -0,0 +1,8 @@
+from lpython import i32
+
+i: i32 = 0
+j: i32
+
+while(i<5):
+    j = i
+    print("j: ", j)
```

Comment:
```suggestion
    print("j: ", j)
    i = i + 1

assert j == 4

```
It seems the variable `i` was not incremented. Thus it leads to an infinite loop. The above change should fix it (Note: I also added an assert).

Reply:
Oops. Didn't realize this :)
Will make the changes quick.